### PR TITLE
adding tramoxadone and supporting chems for post mortem brute restoration

### PR DESCRIPTION
**Pull request by neuPanda**
## About the PR
Adds 1 new chem to create a new cryo chem called Tramoxadone which restores brute damage even if dead

## Why / Balance
This helps balance the changes from med where there are no longer re-stocks and encourage advance med rather than treating everything with brute packs.  This is the Brute version of Aloxodone

## How to test
take a urist,  whap it with a throngler 8 times tossed it into a cryo system and once cooled added the tramoxadone to ensure the brute was cured

**Changelog**
:cl:
- add: new Cryo Chem

### CODE REVIEW

File: Resources/Locale/en-US/_NF/reagents/meta/chemicals.ftl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+reagent-name-salicylicacid = Salicylic Acid
+reagent-desc-salicylicacid = A powdery substence used for dermalogical treatments.
```
Comment (whatston3) [inline Resources/Locale/en-US/_NF/reagents/meta/chemicals.ftl:2]:
```suggestion
reagent-desc-salicylicacid = A powdery substance used for dermatological treatments.
```

Fix typos.

File: Resources/Locale/en-US/_NF/reagents/meta/medicine.ftl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+reagent-name-tramoxadone = tramoxadone
+reagent-desc-tramoxadone = A cryogenics chemical. Used to treat severe trama via regeneration of the damaged tissue. Works regardless of the patient being alive or dead.
```
Comment (whatston3) [inline Resources/Locale/en-US/_NF/reagents/meta/medicine.ftl:2]:
```suggestion
reagent-desc-tramoxadone = A cryogenics chemical. Used to treat severe trauma via regeneration of the damaged tissue. Works regardless of the patient being alive or dead.
```

Fix typo.

**Comment by blueDev2** [comment]
So Salicylic Acid's only use is to react to make Tramoxadone?

**Comment by neuPanda** [comment]
> So Salicylic Acid's only use is to react to make Tramoxadone?

currently yes.

File: Resources/Locale/en-US/_NF/reagents/meta/chemicals.ftl [hunk 3]
```diff
@@ -0,0 +1,2 @@
+reagent-name-salicylicacid = Salicylic Acid
```
Comment (whatston3) [inline Resources/Locale/en-US/_NF/reagents/meta/chemicals.ftl:1]:
```suggestion
reagent-name-salicylicacid = salicylic acid
```
For consistency with other entries in the guidebook.  Missed this at first, sorry.

**Review by whatston3 — COMMENTED** [review]
Ran a bit of a test.  Worked as expected, small change requested on salicylic acid name.

The use case here seems niche.  Having a cryo pod and hotplate setup (sure, you're running a Caduceus), dealing with a body with a variety of brute damage types (extremely violent assault with corpse desecration?), and trying to save on topicals without using sutures?  If space fauna would occasionally attack a player even when dead, stopping before they gib, I could see this being more useful.

One other thing I could think of that would incentivize usage is to reduce the metabolism amount like with Pyrazine, so you conserve more materials.

Looking forward to some goofy use of salicylic acid in the future.

**Review by whatston3 — APPROVED** [review]

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**